src/cli/kfc.ts

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { execSync } from "child_process";
+import prompt from "prompts";
+
+import { RootCmd } from "./root";
+
+export default function (program: RootCmd) {
+  program
+    .command("kfc crd [source] [dest]")
+    .description("Generate class to handle CRD")
+    .action(async (source, dest) => {
```

Comment:
This does not work as expected. `src` is crd

![image](https://github.com/defenseunicorns/pepr/assets/1096507/0bf3618a-fdd2-41a5-91f4-a6be21a216d4)